did/plugins/public_inbox.py

```diff
@@ -0,0 +1,285 @@
+# coding: utf-8
+"""
+Public-Inbox stats about mailing lists threads
+
+Config example::
+
+    [wiki]
+    type = public_inbox
```

Comment:
So far all plugin names are one word only so this will be the first one with a separator. I'd probably prefer `public-inbox` a little bit more but this would need changes in plugin detection. Not sure if it's worth it. Or should we go just with `inbox`? Probably not, as it would be ambiguous... @lukaszachy, any thoughts?

Reply:
IMO dash is nicer as a delimiter than the underscore and there might be another two (or more) word plugins in the future so investing time into the detection could be worth it. And it can be as simple as replace all '_' with`-`. We should be able  to avoid name conflicts (at least for public plugins)

Reply:
Ok, agreed, let's use `public-inbox` in the config. Thanks for the feedback, @lukaszachy. @mripard, will you take care of the underscore substitution or shall I?

Reply:
Whatever is easier for you :)

Given that you already have some additional patches on top, it might be easier to just squash that change into the commit and merge it?

Reply:
I've updated the plugin name to `public-inbox` as part of the new version.